internal/alloycli/automemlimit_cgroups_unsupported.go

```diff
@@ -0,0 +1,25 @@
+//go:build !linux
+// +build !linux
```

Comment:
Apparently cgroups are a Linux feature and not so much a universal UNIX thing, so this should be fine.